### PR TITLE
Adds cross account backup tag to RDS if production

### PR DESCRIPTION
**Pull request by uptickmetachu**
Also fixes 2 broken tests.

### CODE REVIEW

File: gitops_server/deploy.py [hunk 1]
```diff
@@ -114,16 +114,15 @@ async def update_app_deployment(self, app: App):
...
-                    ' --install'
+                await run(
+                    'helm upgrade --install'
                     f' -f {cfg.name}'
                     f" --namespace={app.values['namespace']}"
                     f' {app.name}'
-                    f' {app.chart.helm_chart}'
-                    f' --version={app.chart.version}' if app.chart.version else ''
-                ), catch=True)
+                    f' {app.chart.helm_chart} {chart_version_arguments}', catch=True)
```
Comment (jarekwg) [inline gitops_server/deploy.py:124]: superfluous space bw `{app.chart.helm_chart}` and `{chart_version_arguments}`?

**Review by jarekwg — COMMENTED** [review]
So we need to be prudent with setting the `production` tag on initial creation, because we don't have any mechanisms to set it after the fact?

**Comment by uptickmetachu** [comment]
Yes. We do need to be careful with the production tag. May need a correlation script to check what is backed up.

**Comment by jarekwg** [comment]
Mm, that'd be nice, a library of scripts that we can run to review consistency (in cases where we can't have live checks).

**Pull request merged**